src/lib.rs

```diff
@@ -842,7 +842,7 @@ impl Instruction {
         .unwrap()
         .clone();
 
-        let formatted_name = self.name.replace('_', ".");
+        let formatted_name = self.name.to_owned();
         fmt = fmt.replace("$name$", &formatted_name);
```

Comment:
replacing this with
`fmt = fmt.replace("$name$", &self.name)`
makes it soo the previous line can be omitted